nxrm-aws-ha-helm/values.yaml

```diff
@@ -30,7 +30,7 @@ statefulset:
   container:
     image:
       repository: sonatype/nexus3
-      tag: 3.50.0
+      tag: latest
```

Comment:
When we publish are these replaced with the current version of Nexus?

Reply:
I thought we said we wanted these to be always latest?

Reply:
Hmm,  maybe that was the sample yamls (not helm)

Reply:
Oh for open shift we need to publish a specific [version](https://github.com/sonatype/operator-nexus-repository/blob/a15021fce10d0741050ddeb19317f0b9f59ddd34/helm-charts/sonatype-nexus-repository/Chart.yaml#L24) as it needs to be re-certfied

Reply:
I have also updated the ones in this repo to a specific version. This gets updated by jenkins during release. That reminds me that we'll need to update jenkins jobs for this new helm